src/tests/Myr.test.js

```diff
@@ -880,4 +881,18 @@ describe("Other Myr functionality", () => {
         expect(myr.cursor.color).toEqual("red");
     });
 
+
+    it("setTransparency should set the appropriate cursor attribute correctly", () => {
```

Comment:
We should always test for the negatives as well (aka make sure stuff fails properly).